ts/Core/Axis/Axis.ts

```diff
@@ -271,49 +404,93 @@ class Axis {
         userMax?: number;
         userMin?: number;
     };
+    /** @internal */
     public opposite?: boolean;
+    /** @internal */
     public options!: (AxisOptions|XAxisOptions|YAxisOptions);
+    /** @internal */
     public ordinal?: AxisComposition['ordinal'];
+    /** @internal */
     public overlap!: boolean;
+    /** @internal */
     public paddedTicks!: Array<number>;
+    /** @internal */
     public plotLinesAndBands!: Array<PlotLineOrBand>;
+    /** @internal */
     public plotLinesAndBandsGroups!: Record<string, SVGElement>;
+    /** @internal */
     public pointRange!: number;
+    /** @internal */
     public pointRangePadding!: number;
+    /** @internal */
     public pos!: number;
+    /** @internal */
     public positiveValuesOnly!: boolean;
+    /** @internal */
     public reserveSpaceDefault?: boolean;
+    /** @internal */
```

Comment:
[reversed](https://api.highcharts.com/class-reference/Highcharts.Axis#reversed) is not internal